pkg/controller/dmcluster/dm_cluster_controller.go

```diff
@@ -44,12 +45,14 @@ type Controller struct {
 
 // NewController creates a dmcluster controller.
 func NewController(deps *controller.Dependencies) *Controller {
+	suspender := suspender.NewSuspender(deps)
```

Comment:
Is it OK to move this into the deps?

Reply:
suspender is used only for member managers, so I think don't move into deps